python/ray/tests/test_autoscaler_aws.py

```diff
@@ -0,0 +1,316 @@
+from datetime import datetime
```

Comment:
How would you feel about pulling `test_ausocaler_aws.py` apart into a few files in `python/ray/tests/autoscaler/aws/*.py`?

We've just got a lot of mixed concerns throughout this file:
* Default test data
* Stub setup
* Actual tests

Do you think it might be a bit more approachable if we broke it up into a few files?

Reply:
Agreed - I'll pull it apart a bit. This should set us up better for future growth of the test suite as well.